src/Compilers/CSharp/Portable/Symbols/MemberSignatureComparer.cs

```diff
@@ -140,20 +153,21 @@ internal sealed class MemberSignatureComparer : IEqualityComparer<Symbol>
             considerCallingConvention: false,
             considerRefKindDifferences: true,
             considerArity: false,
-            typeComparison: TypeCompareKind.ObliviousNullableModifierMatchesAny);
+            typeComparison: TypeCompareKind.AllIgnoreOptions);
 
         /// <summary>
-        /// This instance is used to determine if a partial method implementation matches the definition,
-        /// including differences ignored by the runtime.
+        /// Determines if an interceptor has a compatible signature with an interceptable method, including type differences ignored by the runtime.
```

Comment:
The comment may need to be tweaked since some type differences ignored by the runtime are also ignored by this comparer.